.travis.yml

```diff
@@ -59,52 +59,13 @@ script:
 matrix:
     exclude:
         - os: osx
-          env: TASK=documentation
+    include:
         - os: osx
-          env: TASK=lint
+          env: TASK=check-pypy
         - os: osx
-          env: TASK=check-format
+          env: TASK=check-py27
         - os: osx
-          env: TASK=check-quality
-        - os: osx
-          env: TASK=check-coverage
-        - os: osx
-          env: TASK=check-unicode
-        - os: osx
-          env: TASK=check-ancient-pip
-        - os: osx
-          env: TASK=check-py273
-        - os: osx
-          env: TASK=check-py34
-        - os: osx
-          env: TASK=check-py35
-        - os: osx
-          env: TASK=check-pytest28
-        - os: osx
-          env: TASK=check-fakefactory052
-        - os: osx
-          env: TASK=check-fakefactory053
-        - os: osx
-          env: TASK=check-fakefactory060
-        - os: osx
-          env: TASK=check-django17
-        - os: osx
-          env: TASK=check-django18
-        - os: osx
-          env: TASK=check-django19
-        - os: osx
-          env: TASK=check-django110
-        - os: osx
-          env: TASK=check-examples2
-        - os: osx
-          env: TASK=check-examples3
-        - os: osx
-          env: TASK=check-coverage
-        - os: osx
-          env: TASK=check-format
-        - os: osx
-          env: TASK=lint
-    fast_finish: true
```

Comment:
Yeah, I should have done this in a separate commit/PR, sorry. Removing it is fine though: The semantics of fast\_finish unfortunately make it more or less useless. It only causes the build to fast finish if the only remaining jobs are ones which are allowed to fail (which we don't have).